ludwig/profiling/dataset_profile.py

```diff
@@ -0,0 +1,74 @@
+import time
+from typing import Dict, Tuple, Union
+
+import whylogs as why
+from whylogs.core.proto import ColumnMessage
```

Comment:
This line failed for me (whylogs 0.7.9 installed):
```
tests/ludwig/profiling/test_dataset_profile.py:3: in <module>
    from ludwig.profiling.dataset_profile import (
ludwig/profiling/dataset_profile.py:5: in <module>
    from whylogs.core.proto import ColumnMessage
E   ModuleNotFoundError: No module named 'whylogs.core.proto'
```

After upgrading whylogs (1.1.9), the error goes away.  Probably should add a minimum version to `requirements.txt`

Reply:
Good idea, done.